index.html

```diff
@@ -48,6 +49,159 @@ <h4 class="description-heading">I'am a Software Developer</h4>
+                <div class="grid-project-container grid-4">
+                    <h4 class="grid-project-header">Profesional Art Printing Data</h4>
+                    <p class="grid-project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    
+                    <ul class="lang-list-ul">
+                        <li class="lang-list grid-lang-list">html</li>
+                        <li class="lang-list grid-lang-list">bootstrap</li>
+                        <li class="lang-list grid-lang-list">Ruby</li>
+                    </ul>
+                    
+                    <button type="button" class="grid-btn">See Project</button>
+                </div>
+                <!--grid elements 5-->
+                <div class="grid-project-container grid-5">
+                    <h4 class="grid-project-header">Profesional Art Printing Data</h4>
+                    <p class="grid-project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    
+                    <ul class="lang-list-ul">
+                        <li class="lang-list grid-lang-list">html</li>
+                        <li class="lang-list grid-lang-list">bootstrap</li>
+                        <li class="lang-list grid-lang-list">Ruby</li>
+                    </ul>
+                    
+                    <button type="button" class="grid-btn">See Project</button>
+                </div>
+                <!--grid elements 6-->
+                <div class="grid-project-container grid-6">
+                    <h4 class="grid-project-header">Profesional Art Printing Data</h4>
+                    <p class="grid-project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    
+                    <ul class="lang-list-ul">
+                        <li class="lang-list grid-lang-list">html</li>
+                        <li class="lang-list grid-lang-list">bootstrap</li>
+                        <li class="lang-list grid-lang-list">Ruby</li>
+                    </ul>
     
+                    <button type="button" class="grid-btn">See Project</button>
+                </div>
+        </section>    
+
+    <!--About me-->
+    <section class="about">
+        <h1 class="about-header">About me</h1>
+        <p class="about-description">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        <button type="button" class="about-btn">Get My Resume</button>
+        <div class="about-illustration"></div>
+        <hr class="about-line">
+    </section>
+
+    <!--skills-->
+    <section class="skills">
```

Comment:
- [ ] The requirements asked us to create two sections, so kindly add the skills `ul`s to the about section instead of having an individual section for it